src/index.html

```diff
@@ -7,8 +7,35 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <img
+        src="images/logo.png"
+        alt="logo"
+      >
+      </div>
```

Comment:
1. Logo should also be a link to the main page of the website (#home). (CHECKLIST p.8)
2. Also I recommend adding for descriptive `alt`
3. Add class `logo__img` to `<img>` and add sizes in css